src/script/page/LeftSidebar/panels/Conversations/ConversationTabs/ConversationTabs.styles.ts

```diff
@@ -23,10 +23,17 @@ export const footerDisclaimer: CSSObject = {
   display: 'grid',
   gap: '10px',
   gridTemplateColumns: '1fr auto',
+  marginBottom: '4px',
```

Comment:
The spacing between icons are not consistent.
Is it intentional?

<img width="63" alt="Screenshot 2024-08-22 at 10 16 05" src="https://github.com/user-attachments/assets/34cf9820-ddb1-40f8-bb81-aafbe90182a2">

<img width="63" alt="Screenshot 2024-08-22 at 10 15 52" src="https://github.com/user-attachments/assets/24ff9653-8487-4ea4-baee-2c472e91346d">